<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="map_path" default="/home/amsl/hori/vision_nav/map/known_4_actual.pkl"/>
    <arg name="gt_map_path" default="$(find relative_navigator)/topological_maps/dkan_perimeter/known_4_gt_pose.pkl"/>

    <!-- <arg name="map_path" default="$(find relative_navigator)/topological_maps/center_perimeter/unknown_5_actual.pkl"/> -->
    <!-- <arg name="gt_map_path" default="$(find relative_navigator)/topological_maps/center_perimeter/unknown_5_gt_pose.pkl"/> -->

    <!-- <arg name="map_path" default="$(find relative_navigator)/topological_maps/backlight/without_backlight.pkl"/> -->
    <!-- <arg name="gt_map_path" default="$(find relative_navigator)/topological_maps/backlight/without_backlight_gt_pose.pkl"/> -->

    <!-- <arg name="bagfiles_dir" default="/share/private/27th/horiike/bag/abstrelposnet/for_map"/> -->
    <arg name="bagfiles_dir" default="/home/amsl/bag/abstrelposnet/test/actual_c1/proposed"/>
    <arg name="localize_error_output_dir" default="$(find relative_navigator)/localize_error"/>

    <node pkg="relative_navigator" type="topological_map_test_node.py" name="topological_map_test" output="screen">
        <rosparam subst_value="True">
            <!-- hz: 0.1 -->
            hz: 10
            bagfiles_dir: $(arg bagfiles_dir)
            localize_error_output_dir: $(arg localize_error_output_dir)
            map_path: $(arg map_path)
            gt_map_path: $(arg gt_map_path)
            pose_topic_name: "/localized_pose"
            pose_topic_type: "PoseWithCovarianceStamped"
        </rosparam>
    </node>

  <node pkg="rviz" name="rviz_map_frame" type="rviz" args="-d $(find relative_navigator)/config/rviz/map_for_test.rviz" output="screen"/>
</launch>
